gulpfile.js

```diff
@@ -39,7 +39,6 @@ exports.compileIcons = uswds.compileIcons;
 exports.copyFonts = uswds.copyFonts;
 exports.copyImages = uswds.copyImages;
 exports.copyJS = uswds.copyJS;
-exports.copyTheme = uswds.copyTheme;
 exports.copyAssets = uswds.copyAssets;
 exports.updateUswds = uswds.updateUswds;
 exports.buildSass = uswds.compile;
```

Comment:
Minor, but this `buildSass` will call both icon and SASS. Found it by running `npm run uswds:buildSass`. 

```suggestion
exports.buildSass = uswds.compileSass;
```

 --- 

If intentional, we should consider renaming the `buildSass` script in `package.json`.